libraries/eosiolib/contracts/eosio/permission.hpp

```diff
@@ -68,7 +69,7 @@ namespace eosio {
                                    name permission,
                                    const char* pubkeys_data, uint32_t pubkeys_size,
                                    const char* perms_data,   uint32_t perms_size,
-                                   uint64_t delay_us ) {
+                                   microseconds delay_us ) {
      internal_use_do_not_use::check_permission_authorization( account.value, permission.value, pubkeys_data, pubkeys_size, perms_data, perms_size, delay_us );
```

Comment:
I think the `delay_us` argument should be renamed to `delay` and the implementation should be changed to:
```cpp
int64_t delay_us = delay.count();
eosio::check( delay_us >= 0, "negative delay is not allowed" );
internal_use_do_not_use::check_permission_authorization( account.value, permission.value, pubkeys_data, pubkeys_size, perms_data, perms_size, static_cast<uint64_t>(delay_us) );
```

Reply:
got it.